tests/components/onewire/test_entity_owserver.py

```diff
@@ -200,5 +341,6 @@ async def test_owserver_setup_valid_device(hass, device_id):
         assert registry_entry is not None
         assert registry_entry.unique_id == expected_sensor["unique_id"]
         assert registry_entry.unit_of_measurement == expected_sensor["unit"]
+        assert registry_entry.device_class == expected_sensor["class"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)